web/public/templates/dojos/user-profile.dust

```diff
@@ -79,7 +79,7 @@
       <div class="panel panel-default" ng-if="!hideProfileBlock('twitter') || !hideProfileBlock('linkedin')">
         <div class="panel-heading">{@i18n key="Social Media"/}</div>
         <div class="panel-body">
-          <a ng-if="profile.twitter && !hideProfileBlock('twitter')" href="https://{{ profile.twitter }}"><i class="fa fa-twitter fa-2x col-lg-3"></i></a>
+          <a ng-if="profile.twitter && !hideProfileBlock('twitter')" href="https://twitter.com/{{ profile.twitter }}"><i class="fa fa-twitter fa-2x col-lg-3"></i>{{ profile.twitter }}</a>
```

Comment:
The issue with this is that the placeholder on this page https://github.com/CoderDojo/cp-zen-platform/blob/cf2e0c2eaecd208b91fc1bba1e2871e8f155059b/web/public/templates/profiles/general-info.dust#L90 indicates that you need to enter the "twitter.com" part of the URL first. So if I enter "twitter.com/CoderDojo" here now I will end up with "twitter.com/twitter.com/CoderDojo" here. 

I think we need to go with the full solution discussed here https://github.com/CoderDojo/community-platform/issues/695 where we show the twitter.com part of the URL so it is explicit that you need to only enter your username. We'll need to update the placeholder also!